lib/mutant/mutator/node/send.rb

```diff
@@ -92,6 +93,13 @@ def emit_selector_specific_mutations
           emit_lambda_mutation
         end
 
+        def emit_predicate_mutations
+          return unless selector.match?(/\?\z/) && !selector.equal?(:defined?)
```

Comment:
I think we should go for `selector.to_s.end_with?('?')` instead of the regexp.

Rationale: Asking if a string is a suffix is provided natively by `String`. And regexp are a "too powerful tool" for this operation. We should use the simplest API that gets the job done and not use "untapped power" (of regexp in this case).

Reply:
I agree but this was to git around adding an `--ignore-subject`. Later rubies support `end_with?('?')` directly on `Symbol`.

Reply:
I am actually working on adding a mutation to `#start_with?` and `#end_with?` (there is an issue for the former) so this may become an alive again later anyway. How would you like me to resolve this?

Reply:
lets stay with regexp. For the moment, till mutant forces us to think about it again when we add the static suffix and prefix regexp to native method call mutations.